src/06_tuples.py

```diff
@@ -35,6 +35,13 @@ def dist(a, b):
 # Write a function `print_tuple` that prints all the values in a tuple
 
 # YOUR CODE HERE
+def print_tuple(tup):
```

Comment:
Nice job checking for the correct input type! I think it's generally considered more "pythonic" to only use the `range()` method in a for loop when you're not iterating over the entire iterable. It's both shorter and more explicit to just write `for i in tup:`.